cucumber/authenticators_gcp/features/authn_gcp.feature

```diff
@@ -0,0 +1,42 @@
+Feature: GCP Authenticator - Hosts can authenticate with GCP authenticator
+
+  In this feature we define an ￿Google Cloud Platform authenticator in policy and perform authentication
+  with Conjur, using a host with subscription-id & resource-group annotations.
+  In successful scenarios we will also define a variable and permit the host to
+  execute it, to verify not only that the host can authenticate with the Azure
+  Authenticator, but that it can retrieve a secret using the Conjur access token.
+
+  Background:
+    Given a policy:
+    """
+    - !policy
+      id: conjur/authn-gcp
+      body:
+      - !webservice
+
+      - !group apps
+
+      - !permit
+        role: !group apps
+        privilege: [ read, authenticate ]
+        resource: !webservice
+    """
+    And I am the super-user
+    And I have host "test-app"
+
+    And I grant group "conjur/authn-gcp/apps" to host "test-app"
+
+  Scenario: Hosts can authenticate with GCP authenticator and fetch secret
+    Given I have a "variable" resource called "test-variable"
+    And I set service-account-id annotation to host "test-app"
+    And I permit host "test-app" to "execute" it
+    And I add the secret value "test-secret" to the resource "cucumber:variable:test-variable"
```

Comment:
this should go under `I have a "variable" resource called "test-variable"` for context